bittensor/_cli/cli_impl.py

```diff
@@ -413,10 +413,31 @@ def overview(self):
         neurons = []
         block = subtensor.block
         with console.status(":satellite: Syncing with chain: [white]{}[/white] ...".format(self.config.subtensor.network)):
-            for wallet in tqdm(all_hotkeys):
-                nn = subtensor.neuron_for_pubkey( wallet.hotkey.ss58_address )
-                if not nn.is_null:
-                    neurons.append( (nn, wallet) )
+            try:
+                if self.config.subtensor.network not in ('local', 'nakamoto'):
+                    # We only cache neurons for local/nakamoto.
+                    raise Exception("This network is not cached, defaulting to regular overview.")
+            
+                if self.config.no_cache:
+                    raise Exception("Flag was set to not use cache, defaulting to regular overview.")
```

Comment:
Wouldnt the program stop because the config is no_cache when you raise exception here? xd

Reply:
Haha, it's in a try block already.